zebra-chain/src/sapling/commitment.rs

```diff
@@ -164,6 +188,44 @@ impl From<ValueCommitment> for [u8; 32] {
     }
 }
 
+impl std::ops::Sub<ValueCommitment> for ValueCommitment {
```

Comment:
Is there a reason we don't implement `impl<'a> std::ops::Sub<&'a ValueCommitment> for ValueCommitment {` ?

Reply:
Added 👍